fastlane/lib/fastlane/actions/create_pull_request.rb

```diff
@@ -68,6 +68,12 @@ def self.description
         "This will create a new pull request on GitHub"
       end
 
+      def self.output
+        [
+          ['CREATE_PULL_REQUEST_HTML_URL', 'The HTML URL to the created pull request']
```

Comment:
What is a HTML URL?

Reply:
According to [GitHub documentation](https://developer.github.com/v3/pulls/#create-a-pull-request) `html_url` is an URL to the new newly created pull request.